Source/ASTextNode.mm

```diff
@@ -538,64 +538,30 @@ + (UIImage *)displayWithParameters:(id<NSObject>)parameters isCancelled:(NS_NOES
 {
   ASTextNodeDrawParameter *drawParameter = (ASTextNodeDrawParameter *)parameters;
   
-  if (drawParameter->_bounds.size.width <= 0 || drawParameter->_bounds.size.height <= 0) {
+  if (CGRectIsEmpty(drawParameter->_bounds)) {
```

Comment:
Should this be done with a separate diff?

Reply:
I don't know. Should it?